include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -823,6 +823,40 @@ class __static_monotonic_dispatcher<::std::integer_sequence<::std::uint16_t, _X,
     }
 };
 
+// Utility to recommend a stride for the best-performing memory access pattern from empirical testing on different
+// devices. This utility can only be called from the device.
+//
+// SPIR-V compilation targets show best performance with a stride of the sub-group size.
+// Other compilation targets perform best with a work-group size stride.
+template <typename _NdItem>
```

Comment:
Minor thing. I assume there is no need to pass `nd_item` as a template, the function since only `nd_item<1>` is expected.

Reply:
Good point, added!

Reply:
Could you also make the function inline for ODR compliance?

Reply:
Yes, done.